server.go

```diff
@@ -2314,8 +2314,12 @@ func (s *server) peerHandler() {
 	}
 
 	if !cfg.DisableDNSSeed {
+		requiredFeatures := wire.SFNodeNetwork | wire.SFNodeWitness
```

Comment:
I think the node rejects non-segwit peers as sync-peers https://github.com/utreexo/utreexod/blob/affee1685ed328d771dbf01aa1d0a2e13bab84d2/netsync/manager.go#L285-L288

technically segwit isn't necessarily needed and I think looking for those addresses are better than ignoring them.

I guess I'd prefer something like this:

```
requiredServices := defaultRequiredServices
if cfg.Utreexo {
    requiredServices |= wire.SFNodeUtreexo
}
```